pallet-chainlink-feed/src/lib.rs

```diff
@@ -63,6 +63,8 @@ pub mod pallet {
 		pub latest_round: RoundId,
 		pub first_valid_round: Option<RoundId>,
 		pub oracle_count: u32,
+		pub debt: Balance,
+		pub max_debt: Balance,
```

Comment:
I wonder if it makes sense to changes this to an `Option<Balance>` where `None` essentially mean: accumulating debt is not allowed